EpiAware/src/models.jl

```diff
@@ -13,16 +42,17 @@
     neg_bin_cluster_factor ~ neg_bin_cluster_factor_prior
 
     #Latent process
-    time_steps = length(y_t) + n_generate_ahead
+    time_steps = epimodel.time_horizon
     @submodel _I_t, latent_process_parameters =
-        latent_process(data_length; latent_process_priors = latent_process_priors)
+        latent_process(time_steps; latent_process_priors = latent_process_priors)
 
     #Transform into infections
     I_t = transform_function.(_I_t)
 
     #Predictive distribution
     case_pred_dists =
```

Comment:
ultimately can we also express this as a submodel?

Reply:
If you want to shift y_t then yes. Or we can have a function which outputs the predictive distribution if you prefer.